test/scenarios/4. tokens/import erc721 token origin dapp.csv

```diff
@@ -0,0 +1,16 @@
+Step,Test steps,Test data,Expected result,Notes
+1,Open the extension.,,The Welcome Back screen is shown.,
+2,Proceed to Unlock the wallet.,password (8 characters min).,The Ether balance is shown on the overview. The wallet address is shown on the overview. The selected network is Ethereum Mainnet.        ,
+3,Switch networks to a test network.,e.g. Sepolia,The Sepolia balance is shown on the overview. The wallet address is shown on the overview. The selected network is Sepolia.,
+4,Open the test dapp in another tab.,https://metamask.github.io/test-dapp/,,
+5,Proceed to connect with MetaMask. ,,,
+6,Connect with the current account. ,,,
+7,Deploy the NFT contract. ,,,
+8,Confirm the transaction and wait for it to complete.,,,
+9,Proceed to mint NFT. ,,,
+10,Confirm the transaction and wait for it to complete.,,,
+11,Switch to the wallet and open the activity list item.,,"The transaction status, recipient's address, nonce, amount, gas and total are shown in the item details.",
+12,Proceed to view the transaction on the block explorer.,,The block explorer opens in a new tab.,
+13,"In the test dapp, proceed to Watch NFT. ",,,
```

Comment:
Can you add details here to ensure the watch NFT input field is filled in with the ID of the NFT to watch?

Reply:
done!